### DISCUSSION TITLE
Welcome to web3-defi-honeypot-and-slippage-checker Discussions!

### DISCUSSION THREAD

**Discussion by valamidev** [valamidev/web3-defi-honeypot-and-slippage-checker#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Steve4732** [comment]
Hi Valamidev
Thanks for your honeypot checker contract !
Been trying to use it lately.. But i keep on getting 'execution reverted' on the honeyCheck function call for contract 0xe7e07a2281f1e66e938ae7feefc69db181329f12
Was wondering if you could help me find out what is wrong ?

**Comment by Steve4732** [comment]
I'm on Python, here is my function. The eth.call throws 'execution reverted'

    def check_honeypot(self, contract=None):
        address = Web3.to_checksum_address('0xbc9a9ac7dc36b1706732374bf632ef39fb6efbc3')
        if not contract: contract = Web3.to_checksum_address('0xe7e07a2281f1e66e938ae7feefc69db181329f12')
        contract_abi = json.load(open("./ABIs/honeypot_contract.json"))
        honeypot_checker = self.w3.eth.contract(address=contract, abi=contract_abi)
  
        data = honeypot_checker.encodeABI(
            fn_name="honeyCheck",
            args=[self.simu.token_address, self.simu.router_address]
        )
        honeycheck = self.simu.w3.eth.call({
            "from": address,
            "to": contract,
            "data": data,
            "value": int(1 * (10 ** 18)),
        })

**Reply by Steve4732** [reply]
on ETH mainnet
honeycontract 0xe7e07a2281f1e66e938ae7feefc69db181329f12
the token for the test is USDT 0xdac17f958d2ee523a2206206994597c13d831ec7
and the router is Uniswap v2 0x7a250d5630b4cf539739df2c5dacb4c659f2488d
